caffeinated/admin/extend/events/Extend_Events_Admin_Page.core.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 use EventEspresso\core\domain\services\admin\events\default_settings\AdvancedEditorAdminFormSection;
-use EventEspresso\core\domain\services\admin\events\editor\AdvancedEditorData;
+use EventEspresso\core\domain\services\admin\events\editor\EventEditor;
```

Comment:
Yeeaaaaaahhhhhh! It's just an editor, nothing advanced! Why to scare away people? 😄